frontend/src/metabase-lib/queries/utils/card.js

```diff
@@ -2,6 +2,7 @@ import _ from "underscore";
 import { updateIn } from "icepick";
 
 import Utils from "metabase/lib/utils";
+import { getParameterMappingOptions } from "metabase/parameters/utils/mapping-options";
```

Comment:
I need either to move applyParameters to metabase or pass a callback

Reply:
@ranquild which approach would you recommend?

Reply:
`metabase/parameters/utils` should be moved to `metabase-lib`. There is a folder for parameters

Reply:
maybe not everything should be moved, use your own judgement